templates/common/infra/bicep/core/security/keyvault.bicep

```diff
@@ -3,8 +3,12 @@ param name string
 param location string = resourceGroup().location
 param tags object = {}
 
+@description('The ID of the principal to grant access to the key vault.')
```

Comment:
Please add these changes to AVM if needed.  We are deprecating infra/core.

Reply:
Here's the issue: https://github.com/Azure/azure-dev/issues/3327
 
Here's the PR:https://github.com/Azure/azure-dev/pull/3976